src/components/Header/index.tsx

```diff
@@ -0,0 +1,19 @@
+import * as S from './Header.style';
+
+const Header = () => {
+  return (
+    <S.HeaderWrapper>
+      <S.LogoName>Mentors</S.LogoName>
+      <S.BtnWrapper>
+        <S.HeaderBtn>
+          <img src='/img/ProfileBtn.png' alt='' />
+        </S.HeaderBtn>
+        <S.HeaderBtn>
+          <img src='/img/LogoutBtn.png' alt='' />
+        </S.HeaderBtn>
+      </S.BtnWrapper>
+    </S.HeaderWrapper>
```

Comment:
view단에서는 코드를 조금 나눠도 좋을 것 같아요
```suggestion
    <S.HeaderWrapper>
      <S.LogoName>Mentors</S.LogoName>
      
      <S.BtnWrapper>
        <S.HeaderBtn>
          <img src='/img/ProfileBtn.png' alt='' />
        </S.HeaderBtn>
        
        <S.HeaderBtn>
          <img src='/img/LogoutBtn.png' alt='' />
        </S.HeaderBtn>
      </S.BtnWrapper>
    </S.HeaderWrapper>
```